src/ADNLPProblems/hs322.jl

```diff
@@ -0,0 +1,15 @@
+export hs322
+
+function hs322(; n::Int = default_nvar, type::Val{T} = Val(Float64), kwargs...) where {T}
+  function f(x)
+    return (x[1]-20)^2 + (x[2]+20)^2
+  end
+  function c!(cx,x)
+    cx[1] = x[1]^2/100+x[2]^2/0.01-1
```

Comment:
I haven't check the book, but that would be equivalent no?
```suggestion
    cx[1] = x[1]^2/100+x[2]^2 * 100-1
```